sql/core/src/test/resources/sql-tests/results/group-by-filter.sql.out

```diff
@@ -233,14 +233,7 @@ org.apache.spark.sql.AnalysisException
   "sqlState" : "42000",
   "messageParameters" : {
     "expression" : "\"a\""
-  },
-  "queryContext" : [ {
-    "objectType" : "",
-    "objectName" : "",
-    "startIndex" : 56,
-    "stopIndex" : 65,
-    "fragment" : "GROUP BY b"
-  } ]
```

Comment:
Why does the context disappear completely?

Reply:
Previously it is set as `analyzed.origin` which is wrong. It should point to `a` instead of `GROUP BY b` anyway.